hack/placement.yml

```diff
@@ -0,0 +1,7 @@
+kind: ConfigMap 
+apiVersion: v1 
+metadata:
+  name: placements
+data:
+  PLACEMENTS: placement-1st
+  PLACEMENTS_NAMESPACE: default
```

Comment:
Why do we need this ConfigMap? The controller takes environment variables, and soon a CRD Config (#68)

Reply:
I was told we need a placement yaml. Is it irrelevant after the recent changes?

Reply:
It will be irrelevant once #68 is merged yeah